ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml

```diff
@@ -142,6 +142,35 @@ StackLayout {
         }
     }
 
+    Component {
+        id: disabledSettingsBannerComp
+        StatusInfoBoxPanel {
```

Comment:
The layout is a little bit different in Figma. The width is static, to 560 and height dynamic based on content.

App:
<img width="1728" alt="Screenshot 2023-07-24 at 18 07 27" src="https://github.com/status-im/status-desktop/assets/47811206/214a45dc-7d75-4c89-aa70-31f80310503a">

Design:
<img width="1440" alt="Tokens" src="https://github.com/status-im/status-desktop/assets/47811206/aae454fd-3afb-4ef5-bd55-3e480235e292">

Reply:
Fixed, nice spot! (also fixed the padding/radius)